tsconfig.json

```diff
@@ -21,9 +21,6 @@
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
 
-    /* Module Resolution Options */
-    "moduleResolution": "node",
```

Comment:
5.2 complains when this doesn't match the `module: node16` we inherit from `@tsconfig/node14`